core/micro-services/workflow-core/src/main/scala/edu/uci/ics/amber/storage/model/DatasetFileDocument.scala

```diff
@@ -0,0 +1,83 @@
+package edu.uci.ics.amber.storage.model
```

Comment:
All modules in workflow-core should be under amber.core package. If necessary, you could have sub packages but it should be under core

Reply:
moved